src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { VerifiedIcon } from 'nft/components/icons'
+import { CollectionInfoForAsset, GenieAsset } from 'nft/types'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+
+const MAX_WIDTH = 560
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: ${({ theme }) => `calc(100vh - ${theme.navHeight}px)`};
+  align-items: center;
+  padding: 22px 20px 0px;
+  gap: 26px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    gap: 64px;
+    padding-top: 28px;
+  }
+
+  @media screen and (min-width: ${BREAKPOINTS.xl}px) {
+    flex-direction: row;
+    padding-top: 0px;
+    padding-bottom: ${({ theme }) => `${theme.navHeight}px`};
+    gap: 80px;
+  }
+`
+
+const InfoContainer = styled(Column)`
+  width: 100%;
+  align-items: center;
+  gap: 40px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    width: ${MAX_WIDTH}px;
+  }
+`
+
+const InfoDetailsContainer = styled(Column)`
+  gap: 4px;
+  align-items: center;
+`
+
+const MediaContainer = styled.div`
+  width: 100%;
+  height: 100%;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    width: ${MAX_WIDTH}px;
+    width: ${MAX_WIDTH}px;
```

Comment:
height? or duplicate?

Reply:
Also if you use height, be aware of assets with a profile aspect ratio such as `/nfts/asset/0x7ea3cca10668b8346aec0bf1844a49e995527c8b/1036`

Reply:
sorry it was supposed to be height - also yeah the profile sized aspect ratios work because this is just the container